src/OrchardCore/OrchardCore.ContentManagement.Abstractions/ContentQueryExtensions.cs

```diff
@@ -4,14 +4,30 @@
 
 namespace OrchardCore.ContentManagement
 {
+    /// <summary>
+    /// Allows to execute an <see cref="IQuery{ContentItem}"/> to return matching <see cref="ContentItem"/>
```

Comment:
```suggestion
    /// Allows to execute an <see cref="IQuery{ContentItem}"/> to return matching <see cref="ContentItem"/>s
```

Similarly for the others too.

Reply:
Done